stdlib/public/core/Arrays.swift.gyb

```diff
@@ -1694,8 +1694,7 @@ internal func _arrayOutOfPlaceReplace<B, C>(
 /// A _debugPrecondition check that `i` has exactly reached the end of
 /// `s`.  This test is never used to ensure memory safety; that is
 /// always guaranteed by measuring `s` once and re-using that value.
-// FIXME(ABI)#15 : add argument labels.
-internal func _expectEnd<C : Collection>(_ i: C.Index, _ s: C) {
+internal func _expectEnd<C : Collection>(of s: C, is i: C.Index) {
```

Comment:
I thought about doing that, but it didn't sit right with me. assert-like things seem to feel a bit better as stand-alone functions than methods. Not sure I'm being rational about that tho.